surefire-booter/src/main/java/org/apache/maven/surefire/booter/spi/SurefireMasterProcessChannelProcessorFactory.java

```diff
@@ -119,4 +129,23 @@ private final void setTrueOptions( SocketOption<Boolean>... options )
             }
         }
     }
+
+    private UUID parseAuth( URI uri )
```

Comment:
static ?

Reply:
Yes, i will make it static.

Reply:
The method should be renamed because nothing is parsed, you have a `URI` instance already. It is a matter of calling a getter.

Reply:
@eolivelli done!